src/config/rcxml.c

```diff
@@ -1223,6 +1227,8 @@ rcxml_init(void)
 	has_run = true;
 
 	rc.placement_policy = LAB_PLACE_CENTER;
+	rc.placement_cascade_offset_x = 40;
+	rc.placement_cascade_offset_y = 30;
```

Comment:
Should `x` really be different to `y` by default? I don't know anything about how this feature works for other implementations but it feels kind of off to me when the values aren't the same.

Reply:
Personally, I would have `y > x` if I had a vertical monitor so as is is ok IMHO for horizontal orientation which would be the majority of use cases.

Reply:
The settings for the titlebar height (for example a font size 20 for (in)active window) may also play a role here.

Reply:
KWin sets the offset to 1/48 of the usable area ([link](https://github.com/KDE/kwin/blob/df9f8f8346b5b7645578e37365dabb1a7b02ca5a/src/placement.cpp#L364-L368)). So maybe we can allow `%` in the settings to set the offsets relative to the size of usable area.

Another alternative is set the offset to title bar height (perhaps x1.2 or something of it to make it easier to distinguish windows without custom border color or drop shadows) by default and use it for both x and y.